openrr-teleop/src/move_base.rs

```diff
@@ -116,4 +116,76 @@ mod tests {
         assert!(!node.is_enabled);
         assert!(!node.is_turbo);
     }
+
+    #[test]
+    fn test_move_node_get() {
+        let mode = String::from("tested");
+        let base = DummyMoveBase::new();
+        let node = MoveBaseNode::new(mode.clone(), base);
+
+        let base_mode = node.mode();
+        assert_eq!(base_mode, mode);
+        let base_sub_mode = node.submode();
+        assert_eq!(base_sub_mode, &String::from(""));
+    }
+
+    #[tokio::test]
+    async fn test_move_node_proc() {
+        let mode = String::from("tested");
+        let node = MoveBaseNode {
+            move_base: DummyMoveBase::new(),
+            mode: mode.clone(),
+            submode: "".to_string(),
+            vel: BaseVelocity::default(),
+            is_enabled: false,
+            is_turbo: false,
+        };
+        node.proc().await;
+        assert_eq!(
+            format!("{:?}", node.vel),
+            format!("{:?}", BaseVelocity::default())
+        );
```

Comment:
`node.vel` will not be updated regardless of the values of `is_enbled` and `is_turbo`.
I suppose it's better to set node.vel to non-zero first and compare with return value of `node.move_base.current_velocity()` after calling `proc()`.
And, when comparing, it might be better to compare each field with `assert_approx_eq` macro rather than to compare formatted strings.
The same below.

Reply:
- Using `assert_approx_eq`
It is ok. I'll apply.

- `node.vel` will not be updated
I think if `is_turbo` is true, `proc` works `vel` twice and others can't affect.
So, I tested these boolean pattern.